demisto_sdk/commands/validate/validators/IM_validators/IM112_readme_image_too_large.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+import os
+from pathlib import Path
+
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+def is_image_size_valid(content_item: Path) -> bool:
+    doc_files_path: str = f"{content_item}/doc_files"
+    for image_name in os.listdir(doc_files_path):
+        image_path = Path(f"{doc_files_path}/{image_name}")
+        if os.path.isfile(image_path):
+            if os.path.getsize(image_path) > 2 * 1024 * 1024:
+                return False
```

Comment:
Add the bad file name to the error. Currently, there's no way to know which one is too large.